docs/rhel4edge_iso.md

```diff
@@ -168,6 +168,7 @@ virt-install \
     --os-type generic \
     --events on_reboot=restart \
     --cdrom ./microshift-installer-*.$(uname -i).iso \
+    --wait \
```

Comment:
Would it be ok to add a time value (in minutes) to this ? I do not know what the default is if you do not include a time value. I have been using --wait 5, and it works ok. Thoughts ?

Reply:
The manual states:
```
--wait [WAIT]         Minutes to wait for install to complete.
```
So, the wait timeout is optional. I think we should just wait indefinitely until the process completes or fails. 
Picking the "right" timeout is always tricky and I'm not sure it's necessary in this case.